avocado/plugins/multiplex.py

```diff
@@ -35,12 +35,10 @@ def __init__(self, *args, **kwargs):
         super(Multiplex, self).__init__(*args, **kwargs)
 
     def configure(self, parser):
-        if multiplexer.MULTIPLEX_CAPABLE is False:
-            return
-
         parser = super(Multiplex, self).configure(parser)
-        parser.add_argument('multiplex', nargs='+',
-                            help='Path(s) to a multiplex file(s)')
+        if multiplexer.MULTIPLEX_CAPABLE:
+            parser.add_argument("-m", '--multiplex', nargs='*',
```

Comment:
OK.  My concern was that `--mux-inject` is "forcefully" set in the code, and not as an independent source.  The issues with multiple independent sources that can arise are, for example, ordering.  Which "feeder"  should kick in first? The order of calling will give different results after the merge, right?